src/main/java/seedu/duke/Parser.java

```diff
@@ -86,9 +104,13 @@ public static Command checkAction(String inputCommand) throws DukeExceptions {
                 System.out.println("Not in Task View!"); //----------REPLACE WITH EXCEPTION
             } else {
                 for (int i = 1; i < inputs.length; i++) {
-                    projectDescription += inputs[i];
+                    if (i == inputs.length - 1) {
+                        description += inputs[i];
+                    } else {
+                        description += inputs[i] + " ";
+                    }
                 }
-                commandType = new TaskCommand(projectDescription, projectIndex);
+                commandType = new TaskCommand(description, projectIndex);
```

Comment:
Same here. I feel like we shouldn't use variable "description" for both name and description as it will be confusing to use one variable for two different purposes.